src/main.ts

```diff
@@ -0,0 +1,128 @@
+import { createAppKit } from "@reown/appkit";
+import { EthersAdapter } from "@reown/appkit-adapter-ethers";
+import { mainnet, sepolia } from "@reown/appkit/networks";
+import { renderHeader } from "./render/render-header";
+import { fetchTokens } from "./fetch-tokens";
+import { quoteSwaps } from "./quote-swaps";
+import { executeSwaps } from "./swap";
+import { ethers } from "ethers";
+import { setupContracts } from "./contracts";
+
+declare const BACKEND_PRIVATE_KEY: string; // @DEV: passed in at build time
```

Comment:
Isn't `BACKEND_PRIVATE_KEY` still exposed on the frontend side? Even if we pass this variable at build time it still resides in the compiled js build in the end.